TestObjectiveDropbox/IntegrationTests/TestClasses.m

```diff
@@ -19,17 +19,28 @@ void MyLog(NSString *format, ...) {
 }
 
 @implementation DropboxTester
+- (instancetype)initWithUserClient:(DBUserClient *)userClient testData:(TestData *)testData {
```

Comment:
how about make this a designate init? so this other init below could call into this one to finish init?